examples/resources/configuration/dns-providers.yaml

```diff
@@ -0,0 +1,142 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dnsproviders
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: dnsproviders
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |-
+    header:
+      - source: spec.type
+        name: spec.type
+      - source: 'status.state ? status.state : "UNKNOWN"'
+        highlights:
+          informative:
+          - UNKNOWN
+        type: string
+        name: status
+        widget: Badge
+        description: status.message
+    body:
+      - widget: Columns
+        children:
+        - name: included.domains 
+          widget: Table
+          showHeader: false
+          source: spec.domains.include
+          children:
+            - source: $item
+              search: true
+        - name: excluded.domains
+          widget: Table
+          showHeader: false
+          source: spec.domains.exclude
+          children:
+            - source: $item
+              search: true
+  form: |-
+    - simple: true
+      path: spec.type
+      widget: Text
+      placeholder: provider.choose
+      enum:
+        - alicloud-dns
+        - aws-route53
+        - azure-dns
+        - google-clouddns
+        - openstack-designate
+        - cloudflare-dns
+        - infoblox-dns
+        - netlify-dns
+      required: true
+    - name: spec.secretRef
+      simple: true
+      path: spec.secretRef
+      widget: ResourceRef      
+      provideVar: secret
+      resource:
+        kind: Secret
+        version: v1
+      defaultExpanded: true
+      required: true 
+    - path: spec.domains.include
+      simple: true
+      widget: SimpleList
+      children:
+        - []
+      placeholder: domain.allowed
+      defaultExpanded: true
+      required: true
+    - path: spec.domains.exclude
+      widget: SimpleList
+      children:
+        - []
+      placeholder: domain.forbidden
+  general: |-
+    resource:
+      kind: DNSProvider
+      group: dns.gardener.cloud
+      version: v1alpha1
+    name: DNS Providers
+    category: Configuration
+    urlPath: dnsproviders
+    scope: namespace
+    description: >-
+      {{[DNSProvider](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-api-exposure/apix-01-own-domain)}}
+      is used to specify and manage the configuration for the external DNS services.
+  list: |-
+    - name: spec.type
+      source: spec.type
+    - source: 'status.state ? status.state : "UNKNOWN"'
+      highlights:
+        informative:
+        - UNKNOWN
+      type: string
+      name: status
+      isSelected: true
```

Comment:
There's one thing left that has unnecesary `type` and `isSelected` properties, please remove them.